Add example app guides (#832)
supertokens/docs: v2/guides/blocks/recipe-mta-frontend-config.mdx

> On the frontend side there are two main aspects that need to be taken into account:

Reviewer: ```suggestion
```

> - Telling the `Multitenancy` recipe how to figure out which tenant is active. This can be done through the `getTenantId` function. 
> In our example app we store the `tenantId` in `localStorage` and retrieve it each time the function gets called.
> You can choose a different storage solution. As long as you return the correct `tenantId` everything will work properly. 

Reviewer: ```suggestion
Tenant discovery is the process in which your application can know which tenant is trying to log in. Amongst several possibilities, there are two common approaches to this: One is using the sub domain as a identifier for a tenant, the second is to ask the user to enter a tenant ID (or any other identifier using which you can derive the tenant ID). Any which way, once we have a way to get the tenant ID, we need to return that in the `getTenantId` callback in the `multitenancy.init` config on the frontend. In our example app, we ask the user to select their tenant from a dropdown before they try and login. Once the user has selected that, we store  we store the in localstorage, and return it from the `getTenantId` function when it's called by our SDK.

If nothing is returned from the `getTenantId`, or if the `getTenantId` is not defined, the default tenant ID of `"public"` will be used for all users who try and log in.
```

> - Exposing the authentication routes based on the `tenantId`. 
> The `Auth.tsx` page exports a component that renders an UI for selecting the tenant, if no `tenantId` has been set.
> Or the actual login UI, after the user has selected their organization.

Reviewer: ```suggestion
```
